README-PoC.md

```diff
@@ -0,0 +1,71 @@
+# GraphQL Gateway PoC
```

Comment:
Add information how to handle subscriptions in that architecture. grpc has a streaming concept so we can mention about that.